public/app/plugins/datasource/influxdb/datasource.test.ts

```diff
@@ -257,125 +241,6 @@ describe('InfluxDataSource Frontend Mode [influxdbBackendMigration=false]', () =
       expect(ds.database).toBe('fallback');
     });
   });
-
```

Comment:
These tests were deleted because we were checking the cases `interpolateQueryExpr` suite.